interface/src/instruction.rs

```diff
@@ -477,6 +477,21 @@ pub enum TokenInstruction {
     ///
     ///   - `&str` The `ui_amount` of tokens to reformat.
     UiAmountToAmount,
+
+    /// Executes a batch of instructions. The instructions to be executed are specified
+    /// in sequence on the instruction data. Each instruction provides:
+    ///   - `u8`: number of accounts
+    ///   - `u8`: instruction data length (includes the discriminator)
+    ///   - `u8`: instruction discriminator
+    ///   - `[u8]`: instruction data
+    ///
+    /// Accounts follow a similar pattern, where accounts for each instruction are
+    /// specified in sequence. Therefore, the number of accounts expected by this
+    /// instruction is variable, i.e., it depends on the instructions provided.
+    ///
+    /// Both the number of accounts and instruction data length are used to identify
+    /// the slice of accounts and instruction data for each instruction.
```

Comment:
nit: can you add that the contained instructions *can't* be the batch instruction?